Helm chart changes

kousik93 opened the pull request:
Added additionalServiceMonitorLabels template incase of using custom ServiceMonitorSelector.
Fixed helm chart file permissions by removing executable bit.

gianrubio commented on helm/alertmanager/templates/servicemonitor.yaml:
```diff
@@ -9,6 +9,9 @@ metadata:
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
     prometheus: {{ .Release.Name }}
+    {{- if .Values.additionalServiceMonitorLabels }}
```
please bump this chart version

gianrubio commented on helm/exporter-coredns/templates/servicemonitor.yaml:
```diff
@@ -7,6 +7,9 @@ metadata:
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
     prometheus: {{ .Release.Name }}
+    {{- if .Values.additionalServiceMonitorLabels }}
```
also this one

gianrubio commented on helm/exporter-kube-controller-manager/templates/servicemonitor.yaml:
```diff
@@ -7,6 +7,9 @@ metadata:
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
     prometheus: {{ .Release.Name }}
+    {{- if .Values.additionalServiceMonitorLabels }}
```
same here

gianrubio commented on helm/exporter-kube-dns/values.yaml:
```diff
@@ -1,3 +1,5 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-
+## Custom Labels to be added to ServiceMonitor
```
same here

gianrubio commented:
thanks @kousik93

kousik93 commented:
Sorry about that. I updated the requirements.yaml and added it to the last commit.
Just a bit curious, why would you want the executable bit set in the helm chart files?
